src/adr/0002-make-taskmanager-distributed.md

```diff
@@ -0,0 +1,23 @@
+# 2. Make TaskManager Distributed
+
+Date: 2019-10-02
+
+## Status
+
+Accepted
```

Comment:
I believe we can add the `LazyConsensus` wording here wich means "we at linagora proposes and contributes this until nobody complains".

Accepted should be kept for agreement from the community (at least a mail on server-dev being answered favorably)

We should also have `Voted: Accepted` and `Voted: Declined` IMO

The wording here needs further rafinement